cli/src/style.rs

```diff
@@ -0,0 +1,75 @@
+//! Style and colouration of Iroha CLI outputs.
+use owo_colors::{OwoColorize, Style};
+
+/// Styling information set at run-time for pretty-printing with colour
+#[derive(Clone, Copy, Debug)]
+pub struct Styling {
+    /// Positive highlight
+    pub positive: Style,
+    /// Negative highlight. Usually error message.
+    pub negative: Style,
+    /// Neutral highlight
+    pub highlight: Style,
+    /// Minor message
+    pub minor: Style,
+}
+
+impl Default for Styling {
+    fn default() -> Self {
+        Self {
+            positive: Style::new().green().bold(),
+            negative: Style::new().red().bold(),
+            highlight: Style::new().bold(),
+            minor: Style::new().green(),
+        }
+    }
+}
+
+/// Determine if message colourisation is to be enabled
+pub fn should_disable_color() -> bool {
+    supports_color::on(supports_color::Stream::Stdout).is_none()
+        || std::env::var("TERMINAL_COLORS")
```

Comment:
Since `supports_color` takes [NO_COLOR](http://no-color.org/) into account, do we really need to read one more env variable?